VISION.md

```diff
@@ -103,6 +105,102 @@ The OpenEBS project and CNCF recognize the importance of supporting this and mai
+
+Listed below are the goals and objectives:
+### 1. Cloud Native Storage for Kubernetes
+   - Ensure seamless integration with Kubernetes environments.
+   - Provide container-attached storage that aligns with Kubernetes-native architectures.
+   - Deliver dynamically provisioned storage with persistent volumes (PVs) managed via Kubernetes StorageClasses.
+
+### 2. Open Source and Community-Driven
+   - Foster an open and active community contributing to continuous innovation.
+   - Ensure transparency, extensibility, and flexibility through an open governance model.
+   - Collaborate with other Cloud Native Computing Foundation (CNCF) projects and Kubernetes ecosystem tools.
+
+### 3. Simplified Storage Management
+   - Enable DevOps teams to provision, manage, and scale storage without requiring deep storage expertise.
+   - Offer policy-driven automation for storage provisioning, data protection, and lifecycle management.
+   - Ensure easy deployment with no complex external dependencies.
+
+### 4. Scalability and Performance Optimization
+   - Support a scale-out architecture, allowing storage resources to grow with application demand.
+   - Provide flexible data replication and storage engine choices (Local and Replicated PV) to match performance needs.
+   - Optimize storage for cloud, hybrid, and on-premises Kubernetes deployments.
+
+### 5. Data Resilience and High Availability
+   - Offer built-in replication, snapshots, and backup capabilities.
+   - Ensure data integrity and protection across nodes and clusters.
+   - Support disaster recovery.
+
+### 6. Developer and DevOps Friendly
+   - Provide a declarative, Kubernetes-native approach to storage management.
+   - Support GitOps, CI/CD pipelines, and automated workflows for persistent storage.
+   - Enable easy monitoring and troubleshooting with Kubernetes observability tools.
+
+### 7. Storage Agnostic and Portable
+   - Work across cloud providers, on-premises, and hybrid environments without vendor lock-in.
+   - Support multiple storage backends, including local disks, networked storage, and cloud block storage.
+   - Integrates with cloud native backup tools to seamlessly migrate application data between different storage types and clusters.
+
+### Differentiation in the Cloud Native Landscape
+Unlike traditional storage solutions, OpenEBS takes a Kubernetes-first approach, allowing:
+  -  **Flexible Storage Choice:** Provide flexible data replication and storage engine choices (Local and Replicated PV) to match performance needs.
+   - **Containerized Storage Architecture:** Runs as Kubernetes pods, making it portable and easy to deploy across different environments.
+   - **Granular Storage Customization:** Developers can fine-tune replication, compression, and storage engine selection on a per-application basis.
```

Comment:
>**Granular Storage Customization:** Developers can fine-tune replication, compression, and storage engine selection on a per-application basis.

This seems a little far fetched

Reply:
App-1 needs replication, use Mayastor
App-2 needs compression, use LocalPV ZFS
How is this far-fetched?